src/components/Categories.js

```diff
@@ -0,0 +1,10 @@
+import '../cat.css';
+
+function Categories() {
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center' }}>
+      <button type="button">Check status</button>
+    </div>
+  );
+}
```

Comment:
- [ ] For this component, you're expected to dispatch the checkStatus action when the button is clicked so the method dispatches the action and updates the categories state, and then renders the Under construction text on the screen.
- [ ] Kindly import the useDispatch hook and create a dispatch variable, then also create a function in which you dispatch the action which is again imported then you pass this function into the button when it's clicked. 👍🏽

Reply:
Fixed.